lib/help-search.js

```diff
@@ -26,28 +27,17 @@ class HelpSearch extends BaseCommand {
 
   async helpSearch (args) {
     if (!args.length)
-      throw this.usage
+      return this.npm.output(this.usage)
```

Comment:
Now that this isn't pulling triple duty it can just work like everything else and return usage on no params